ode/blueprints/amu/birthday_functions.py

```diff
@@ -1,38 +1,24 @@
-#import datetime
 from datetime import date, datetime
-
-from icalendar import Calendar
-
-def calculate_age(birthdate):
-    birthdate = birthdate.strftime("%Y-%m-%d")
-
-    if birthdate != '':
-        birthdate = datetime.strptime(birthdate, "%Y-%m-%d")
-        today = date.today()
-        return (today.year - birthdate.year - ((today.month, today.day) < (birthdate.month, birthdate.day))) + 1
-    return None
-
+import uuid
+import socket
 
 def create_ical(user_list, group_list):    
     event = ''
 
     for user in user_list:
-        summary = 'Birthday from {}'.format(user.name)
+        summary = _('Birthday from {}').format(user.name)
         date = user.birthdate.strftime("%Y%m%d")
         groups = ''
 
         for group in group_list:
             if user.dn in group.members:
                 groups += '{} '.format(group.name)
 
+        uuid = "{}-{}@{}".format(user.name, 'amu', socket.gethostname().lower())
```

Comment:
Sicherheitshalber sollte auch das Geburtstatum mit rein, falls wir sonst später noch andere Termine generieren.

Außerdem bitte `user.name.replace(' ', '-')` – Leerzeichen sind zwar grundsätzlich in UIDs erlaubt, aber unüblich und unschön.